### DISCUSSION TITLE
如何设置监控时间段

### DISCUSSION THREAD

**Discussion by mxlt02** [apache/hertzbeat#64] [Q&A] [answered]
开发者们, 你好.

现有以下使用场景
针对某网站的监控, 该网站的服务器被统一管理, 晚上12点之后统一屏蔽外网
白天我正常的使用, 但到了晚上我会收到一堆警告, 我非常希望使用监控系统在白天, 但晚上可以停止监控

**Comment by tomsun28** [comment, marked as answer]
您好，如您描述需要的功能应该是监控定时挂起维护，后面我们会在晚几个版本开发此功能。   
针对目前的情况，建议将hertzbeat部署到服务器内网，这样就不会有外网问题。

**Comment by tomsun28** [comment]
> 白天我正常的使用, 但到了晚上我会收到一堆警告, 我非常希望使用监控系统在白天, 但晚上可以停止监控

hi你好，针对这个常见需求我们开发了告警静默功能，用户可以配置一次性时间段或周期性时间段内对告警静默，从而解决不想被打扰等问题，欢迎使用
